target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq

```diff
@@ -10,6 +10,12 @@ set_preinit_iface() {
 	TDW8970)
 		ifname=eth0
 		;;
+	TDW8980)
+		ifname=eth0
+		;;
+	TDW9980)
```

Comment:
Do not add code for devices which do not exists. Is very likely that a TWD9980 will never exists, since the hardware is identical to the TWD8980-